main.py

```diff
@@ -253,7 +254,14 @@
             continue
         f = file.split('.')
         if len(f) == 2:
-            contexts[f[0]] = json.loads(open(_C.FORK_DIR + '/' + file).read())
+            if f[0] not in contexts:
+                contexts[f[0]] = {}
+            contexts[f[0]]['results'] = json.loads(open(_C.FORK_DIR + '/' + file).read())
+        elif f[1] == "charts":
+            ## Create and consolidate charts findings
+            if f[0] not in contexts:
+                contexts[f[0]] = {}
+            contexts[f[0]]['charts'] = json.loads(open(_C.FORK_DIR + '/' + file).read())
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the resource returned by the following method call: *open*. The resource is allocated by call *builtins.open*. Currently, there are execution paths that do not contain closure statements, for example, when *TextIOBase.read()* throws an exception. To prevent this resource leak, close the object returned by *open()* in a try-finally block or declare it using a `with` statement.

**More info**
[View details about the `with` statement in the Python developer&#39;s guide](https://www.python.org/dev/peps/pep-0343/) (external link).